.github/workflows/test-make.yml

```
@@ -15,12 +15,15 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref_name }}
   cancel-in-progress: true
 
-env:
-  MAKE_TARGET: ${{ github.event.pull_request.head.repo.fork && 'community' || 'all_ci' }}
-  BAKE_TARGET: ${{ github.event.pull_request.head.repo.fork && 'community' || 'enterprise' }}
-
 jobs:
   make-test:
+    strategy:
+      matrix:
+        target: ${{ fromJSON(
+            github.event.pull_request.head.repo.fork
+              && '["community","search_service","tengines","acc"]'
```

community is already a group containing the subsequents targets: https://github.com/Alfresco/alfresco-dockerfiles-bakery/blob/e394ce2c3bc9963e44508b8d4b76d3cda9c8c546/docker-bake.hcl#L17

```suggestion
              && '["content_service_community","search_service","tengines","acc"]'
```